### DISCUSSION TITLE
Accepting PR

### DISCUSSION THREAD

**Discussion by animex007** [jitacm/-30DaysDevChallenge-#11] [Q&A] [answered]
what are the rule for accepting PR, my PR is not merged till now

**Comment by jitacm** [comment, marked as answer]
it takes some time to review the changes and accept ,plz have patience

**Comment by jitacm** [comment]
your PR will be merges soon !

**Comment by animex007** [comment]
there also conflict in while creating PR on your Repo

**Reply by rohansnishad** [reply]
We are looking on it

**Reply by jitacm** [reply]
plz , check if you have any issue from your end
